src/Commands/MakeEnum.php

```diff
@@ -22,6 +22,11 @@ protected function getDefaultNamespace($rootNamespace)
         return $rootNamespace.'\Enums';
     }
 
+    public function info($string, $verbosity = null)
```

Comment:
This method is used for the mentioned output but could also be used for other output.
This command can also output an already exists message which is done via `error()`.